twitter/streams.go

```diff
@@ -230,21 +231,38 @@ func (s *Stream) retry(req *http.Request, expBackOff backoff.BackOff, aggExpBack
 // scan error, or the done channel is closed.
 func (s *Stream) receive(body io.ReadCloser) {
 	defer body.Close()
-	// A bufio.Scanner steps through 'tokens' of data on each Scan() using a
-	// SplitFunc. SplitFunc tokenizes input bytes to return the number of bytes
-	// to advance, the token slice of bytes, and any errors.
-	scanner := bufio.NewScanner(body)
```

Comment:
Another option is to just increase the `bufio` max token size:
- https://golang.org/src/bufio/scan.go?s=3594:3631#L69

Default is 64k, can bump it up to 512k (we found that worked for us).
```go
	scanner := bufio.NewScanner(body)
 +	scanner.Buffer(nil, 512*1024)
```